jekyll/_includes/segment.html

```diff
@@ -10,7 +11,9 @@
     "title"      : {{ page["title"] | jsonify }},
     "short-title": {{ page["short-title"] | jsonify }},
     "categories" : {{ page["categories"] | jsonify }},
-    "description": {{ page["description"] | jsonify }}};
+    "description": {{ page["description"] | jsonify }},
+    "collection": {{ page.collection | jsonify }}
```

Comment:
@ndintenfass I thought the use of the old notation was strange there; less clean, IMO. But I'll re-do that for consistency.